slave/process-fs-scratch-local/changelog

```diff
@@ -1,3 +1,10 @@
+perun-slave-process-fs-scratch-local (3.1.5) stable; urgency=low
+
+  * Modify service behaviour to be able to continue from it's own failure
+  * Change access control list on scratch directory when there are any differences
```

Comment:
It also changes ownership.
Also, it would be good if you can describe this in detail. This is a significant change in the service behavior, and we should explain it properly.